src/xiaomi_ble/parser.py

```diff
@@ -1381,6 +1384,10 @@ def _parse_xiaomi(
         self.device_id = device_id
         self.device_type = device_type
 
+        # set to True if the device is not sending regular BLE advertisements
+        if device_type in SLEEPY_DEVICE_MODELS:
```

Comment:
You could write this as self.sleepy_device = device_type in …

Reply:
Thx. Changed it.